src/holofuel/HFRouter.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
```

Comment:
I wanted to test all 3 cases for this component because it's the kind of thing that will otherwise get broken in the course of everyday use (we might break one of the modes and not notice for a long time if we're just always using a different mode).

This test is a little annoying, and keeping it up to date (adding new links and assertions as we add new pages to HF) will be a bit of a chore, but I think overall it's worth it.